src/components/filesystem/index.js

```diff
@@ -196,14 +199,43 @@ module.exports = {
             }
         });
             }
+        this.showPendingMessages();
 	},
-	
-        processPending: function() {
-            for (var i=0; i < this.onUpdateCompletion.length; i++) {
-                this.onUpdateCompletion[i].call();
+	showPendingMessages: function() {
```

Comment:
can we call this showPendingServerMessages? I'm trying to keep the naming clear and reserve message on its own for keymail.

Reply:
done